src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/integration/ReferralControllerIntegrationTest.kt

```diff
@@ -785,6 +786,9 @@ class ReferralControllerIntegrationTest : IntegrationTestBase() {
 
   fun getReferralById(createdReferralId: UUID) =
     webTestClient
+      .mutate()
```

Comment:
this fixes the 5 second timeout issue. I know you do this in the parent class too but for some reason that doesn't work but this way does. Just a shame you have to do it each time rather than globally. There will be a way but didn't have time to look